.github/workflows/ci.yaml

```diff
@@ -62,9 +65,9 @@ jobs:
             sudo dpkg -i containernetworking-plugins_1.1.1+ds1-1_amd64.deb
           fi
 
-      - name: (Ubuntu) Install docker compose
-        if: matrix.TEST == 'docker' && matrix.os == 'ubuntu-latest'
-        uses: docker-practice/actions-setup-docker@1.0.11
+      #- name: (Ubuntu) Install docker compose
+      #  if: matrix.TEST == 'docker' && matrix.os == 'ubuntu-latest'
+      #  uses: docker-practice/actions-setup-docker@1.0.11
```

Comment:
Why are these commented out?

Reply:
https://github.com/ansible/galaxy_ng/pull/1800

Reply:
although it says `Install docker compose`, it installs docker and docker-compose. I believe you need this to test oci-env using docker and yet you can install the new `docker-compose` binary.

Reply:
docker is already installed in the ubuntu vms provided by github actions. Also... the tests are passing without this.